sbom-utility-scripts/scripts/add-image-reference-script/README.md

```diff
@@ -0,0 +1,95 @@
+# Add image reference script
+
+The script aims to enrich the SBOM file with additional information about the images used in the build process.
```

Comment:
> information about the images used in the build process

To me this gives the false impression that this script handles the `FROM` images in the Containerfile (there's a different script for that in this repo). "Information about the output image" might be more accurate

Reply:
Fixed